packages/nextjs/src/common/wrapGenerationFunctionWithSentry.ts

```diff
@@ -102,6 +102,8 @@ export function wrapGenerationFunctionWithSentry<F extends (...args: any[]) => a
               attributes: {
                 [SEMANTIC_ATTRIBUTE_SENTRY_SOURCE]: 'route',
                 [SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN]: 'auto.function.nextjs',
+                'sentry.nextjs.function.type': generationFunctionIdentifier,
+                'sentry.nextjs.function.route': componentRoute,
```

Comment:
we need to add these to https://github.com/getsentry/sentry-conventions

Reply:
https://github.com/getsentry/sentry-conventions/pull/54